### ISSUE TITLE
Helper Script for Documentation Aid

### ISSUE THREAD

**Issue by KingGooseJr** [Telmate/terraform-provider-proxmox#777]
tl;dr I would like to provide a bash script that can be referenced in the terraform provider docs as a quick and easy way to install a template onto your proxmox server that can be used with the Telmate/Proxmox Terraform provider.

Would the maintainers of this project be interested in having this as part of the repo?

## Information
- The documentation references Proxmox docs for creating a template for using with the Telmate Terraform provider.
- This documentation can be overwhelming for new users who are trying to get up and going quickly.

## Deliverable
- A helper script that quickly downloads and generates a template or a few templates on the proxmox server for the user to begin playing with.

## Example
- I began [this script](https://github.com/KingGooseJr/Proxmox/blob/main/scripts/cloud-template-create.sh) for my own use, but then started letting it evolve to become a command line tool for installing a template.
- It is a work in progress, but would give you a few interactive options to manage the installation of a template on your proxmox server, or could be run outright (open to suggestions here) in order to quickly put a template onto the proxmox server.

## Considerations
- How interactive do we want it
- Do we want it to clean up older versions of the templates the script can create
- Should it have just one template or use different server/cloud images available
